web/.dependency-cruiser.js

```diff
@@ -206,7 +206,18 @@ module.exports = {
       from: {},
       to: { path: "rehype-raw" },
     },
-
+    {
+      from: {},
+      to: { path: "unist-util-visit" },
+    },
+    {
+      from: {},
+      to: { path: "remark-directive" },
+    },
+    {
+      from: {},
+      to: { path: "@types" },
```

Comment:
I wouldn't expect this to be necessary because you're never importing directly from the @types package... does it work without this one?

Reply:
The customRemarkPlugin function within PureMarkdownContent has `  | void
  | import("unified").Transformer<import("mdast").Root, import("mdast").Root>` as the return type. When this is removed, I get the following error from the dependency check `error not-in-allowed: src/components/PureMarkdownContent.tsx → ../node_modules/@types/mdast/index.d.ts`